src/ruby/ext/symengine/ruby_symbol.h

```diff
@@ -0,0 +1,9 @@
+#include "ruby.h"
+#include "ruby_basic.h"
+#include "cwrapper.h"
+
+void csymbol_free(void *ptr);
```

Comment:
Cool, thanks. Did you try to run the test suite in Valgrind? Just to make sure it doesn't leak.